engines/wage/design.cpp

```diff
@@ -265,7 +265,7 @@ void drawPixelPlain(int x, int y, int color, void *data) {
 }
 
 void Design::drawRect(Graphics::ManagedSurface *surface, Common::ReadStream &in,
-				Patterns &patterns, byte fillType, byte borderThickness, byte borderFillType) {
```

Comment:
Please revert the indentation. It was in purpose for better readability. Same goes for below.